docs/src/main/sphinx/installation/cli.rst

```diff
@@ -358,8 +364,51 @@ The following table list the available options for Kerberos authentication:
   * - ``--krb5-service-principal-pattern``
     - Remote kerberos service principal pattern (default: ${SERVICE}@${HOST})
 
-See :doc:`/security/cli` for more information on configuring and using Kerberos
-with the CLI.
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+Many of the same steps, that can be used when troubleshooting the :ref:`Trino
+coordinator <coordinator-troubleshooting>` apply to troubleshooting the CLI.
+
+Additional Kerberos debugging information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can enable additional Kerberos debugging information for the Trino CLI
```

Comment:
This approach needs to be broken up .. 

basically there are multiple topics here 

- add more JVM flags to run CLI 
- what are the JVM flags for Kerberos debugging
- example of using Kerberos options
please arrange a meeting so I can explain more in detai

Reply:
and we need to cover these in separate sections in this doc

Reply:
Pushed my changes. Will do this in the next commit.